client/my-sites/stats/subscribers-section/index.tsx

```diff
@@ -45,17 +53,35 @@ export default function SubscribersSection( {
 	period?: string;
 } ) {
 	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
-	const quantity = 30;
+	const quantityDefault: QuantityDefaultType = {
+		day: 30,
+		week: 12,
+		month: 6,
+		year: 3,
+	};
+	const quantity = quantityDefault[ period as keyof QuantityDefaultType ];
+	const [ queryDate, setQueryDate ] = useState( new Date() );
```

Comment:
In most other places, we use site timezone, and a couple others use user timezone. I'd lean to always using site timezone at the stage, because the API is based on that. However user or browser timezone makes sense too - I guess we'll need to come to an agreement about this.  🤔 😄 

- Site?
- User?
- Browser?

@jsnmoon